tests/layer_tests/pytorch_tests/test_adaptive_avg_pool.py

```diff
@@ -34,17 +33,17 @@ def forward(self, input_tensor):
     @pytest.mark.precommit
     @pytest.mark.precommit_ts_backend
     @pytest.mark.precommit_fx_backend
-    def test_adaptive_avg_pool3d(self, ie_device, precision, ir_version, input_tensor, output_size):
-        self.input_tensor = input_tensor
+    def test_adaptive_avg_pool3d(self, ie_device, precision, ir_version, input_shape, output_size):
+        self.input_tensor = np.random.random_sample(input_shape).astype(np.float32)
```

Comment:
Why `random_sample`?

Reply:
it accepts list as argument

Reply:
changed with set seed argument, jfyi, it increased the execution time on 1minute

Reply:
You create generator in each test case, which make sense for parallel, but can be the reason for this degradation. And tbh I was thinking we will create just one generator with seed for all test suite. Also it complicates each test, so this actually change my mind, would it be acceptable to revert using generators with seed back?

Reply:
reverted